package.json

```
@@ -5,8 +5,10 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "dev": "nodemon index",
-    "start": "node index"
+    "dev": "nodemon -e ts --exec \"npm run dev:build\"",
+    "start": "node build/index",
+    "build": "tsc",
+    "dev:build": "tsc && node build/index"
```

`postinstall` tells Heroku what to do after installing the npm packages
```suggestion
    "dev:build": "tsc && node build/index",
    "postinstall": "tsc"
```

Also, add a new file to the root directory called Procfile and put in `web: node build/index.js`
This tells Heroku the entry point of our app as documented [here](https://medium.com/developer-rants/deploying-typescript-node-js-applications-to-heroku-81dd75424ce0)